### PR TITLE
Implemented table of contents using TOC from manifest instead of contents link

### PR DESCRIPTION
**Pull request by aslagle**

### CODE REVIEW

File: src/IFrameNavigator.ts [hunk 1]
```diff
@@ -341,14 +373,30 @@ export default class IFrameNavigator implements Navigator {
     }
 
     private handleContentsClick(event: MouseEvent): void {
-        const position = {
-            resource: this.contentsLink.href,
-            position: 0
-        };
-        this.navigate(position);
+        const display: string | null = this.toc.style.display;
+        if (display && display === "none") {
```
Comment (winniequinn) [inline src/IFrameNavigator.ts]: We can just do `display === "none"`.

**Review by winniequinn — CHANGES_REQUESTED** [review]
As discussed, I think it would be better if the TOC were full-screen. It would make it easier to have UI for closing the TOC, would make dealing with accessibility issues somewhat easier, and would allow for a less cramped view on mobile. Our current apps for Android and iOS are a good source of inspiration here.

File: src/IFrameNavigator.ts (hunk 1 above)
Comment (aslagle) [reply src/IFrameNavigator.ts]: fixed

**Comment by aslagle** [comment]
I changed the example in this repo to have a full-screen TOC, and it's scrollable now.

**Review by winniequinn — APPROVED** [review]

**Pull request merged**